lua/rust-tools/external_docs.lua

```diff
@@ -0,0 +1,17 @@
+local M = {}
+local utils = require("rust-tools.utils.utils")
+
+function M.open_external_docs()
+  utils.request(
+    0,
+    "experimental/externalDocs",
+    vim.lsp.util.make_position_params(),
+    function(_, url)
+      if url then
+        vim.fn["netrw#BrowseX"](url, 0)
```

Comment:
I'm not 100% sure this is the best way, online I read much about this not working? 

Or should we make it possible to override this in config/setup?

Reply:
I think this should be fine, since it's a vim thing iirc, and users can set netrw_browsex_viewer to change the browser